### PR TITLE
Sass: Upgrade to Tokens (ids-identity@2.0.0)

### PR DESCRIPTION
**Pull request by clepore**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
We are updating Enterprise to use the new tokens (naming device changed).

**Related github/jira issue (required)**:
n/a

**Steps necessary to review your pull request (required)**:
1. `npm run build`
    - It should build fine and browsing the site should look exactly the same as it did before this PR.

### CODE REVIEW

File: src/components/autocomplete/_autocomplete.scss [hunk 1]
```diff
@@ -62,11 +62,11 @@
   }
 
   .is-selected i {
-    color: $dropdown-selected-color;
+    color: $dropdown-color-selected-font;
   }
 
   // li:hover i {
```
Comment (tmcconechy) [inline src/components/autocomplete/_autocomplete.scss]: Could remove

File: src/components/button/_button.scss [hunk 2]
```diff
@@ -126,59 +126,59 @@ button {
 
 // standard, primary, destructive
 .btn-primary {
-  background-color: $button-primary-bg-color;
-  border: 1px solid $button-primary-bg-color;
-  color: $button-primary-text-color;
+  background-color: $button-color-primary-initial-background;
```
Comment (tmcconechy) [inline src/components/button/_button.scss]: Im finding the names a bit confusing to read. Do we maybe need an explanation added?
i.e. whats the difference between `initial` `base` and `theme` they all seem to imply the same thing and i wonder if there are redundent?

File: src/components/autocomplete/_autocomplete.scss (hunk 1 above)
Comment (clepore) [reply src/components/autocomplete/_autocomplete.scss]: We don't need this style anymore?

File: src/components/button/_button.scss (hunk 2 above)
Comment (clepore) [reply src/components/button/_button.scss]: Yeah I can understand that. We'll put some documentation together for everyone to digest a bit.

Comment (nickwynja) [reply src/components/button/_button.scss]: I raised https://github.com/infor-design/design-system/issues/227 to properly document the tokens. @tmcconechy if you have any additional specific questions, could you ask on that ticket?

**Comment by clepore** [comment]
Okay, so I think this is all updated to our final release versions, and should appease the masses.

**Comment by nickwynja** [comment]
I am appeased.

**Comment by clepore** [comment]
Appeased without 🍌 ...maybe an 🍊 ?

**Comment by nickwynja** [comment]
🍌 🍌

**Comment by nickwynja** [comment]
🍊 you glad I 🍌 'd?

**Comment by clepore** [comment]
2for1

**Pull request merged**